app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -27,13 +27,13 @@ class MainActivity : AppCompatActivity() {
     private lateinit var appBarConfiguration: AppBarConfiguration
     private lateinit var binding: ActivityMainBinding
 
-    val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
+
     val keybindsFragmentActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
     val projectsFragmentActionMenuIds= setOf(R.id.action_add)
     val shareFragmentActionMenuIds=setOf<Int>()
 
-    var Menu:Menu?=null;
-    var onMenuInit:MenuInitialized?=null;
+    var menu:Menu?=null
+    var onMenuInit:MenuInitialized?=null
```

Comment:
Good, but this was already covered by two students in previous Sprint 2 Code Reviews.  When that happens, first in wins.